Rakefile

```diff
@@ -1,22 +1,4 @@
-require 'bundler'
-Bundler.require(:rake)
-require 'rake/clean'
-
-CLEAN.include('spec/fixtures/', 'doc', 'pkg')
-CLOBBER.include('.tmp', '.librarian')
-
-require 'puppetlabs_spec_helper/rake_tasks'
-
-task :default => [:clean, :spec]
-
-PuppetLint.configuration.fail_on_warnings = true
-PuppetLint.configuration.send('relative')
-PuppetLint.configuration.send('disable_80chars')
-PuppetLint.configuration.send('disable_class_inherits_from_params_class')
-PuppetLint.configuration.send('disable_class_parameter_defaults')
-PuppetLint.configuration.send('disable_documentation')
-PuppetLint.configuration.send('disable_single_quote_string_with_variables')
-PuppetLint.configuration.send('disable_empty_string_assignment')
-PuppetLint.configuration.ignore_paths = ["spec/**/*.pp", "pkg/**/*.pp"]
-
+require 'simp/rake/pupmod/helpers'
+require 'puppet-strings/tasks'
```

Comment:
Is this require needed?

Reply:
I copied this in from another module.  Since the skeleton is not up to date, I'm honestly not sure what we require these days.